buildSrc/src/main/java/com/osacky/flank/gradle/FladleConfigImpl.kt

```diff
@@ -25,5 +25,16 @@ data class FladleConfigImpl(
   override var resultsBucket: String?,
   override var keepFilePath: Boolean,
   override var resultsDir: String?,
-  override var additionalTestApks: Map<String, List<String>>
+  override var additionalTestApks: Map<String, List<String>>,
+  override var runTimeout: String? = null,
```

Comment:
btw, i updated the rest of this class, but you can remove the redundant default specification from this file!